### DISCUSSION TITLE
Exclude all folders except required to a User

### DISCUSSION THREAD

**Discussion by khornerz** [filebrowser/filebrowser#1494] [Q&A]
Hi, I've configured filebrowser to see all subfolders from my "NAS" folder.
I've in it
/Movies
/Series
/Music
/Configs
/Documents

I want one user can only see Movies Series and Music.
How can I do these in UI?
I have tried some to do regex but something like: [x] regex  [ ] allow      ^(?!.Movies|.Series|.Music).*
thows me an error.

**Comment by o1egl** [comment]
I would simplify it to a few disallow rules like 
`Documents.*`
`Configs.*`
etc...

**Reply by JustMentalMatt** [reply]
How do you add a disallow rule? the only check boxes are Regex and Allow, and even unchecking Allow doesn't work for me. :(

**Reply by ramiresviana** [reply]
Unchecking the Allow option will make the rule to be processed as a disallow rule. When Regex is not used, you must provide the file/folder full path.

**Reply by JustMentalMatt** [reply]
> Unchecking the Allow option will make the rule to be processed as a disallow rule. When Regex is not used, you must provide the file/folder full path.

Hmm, maybe im doing something wrong then. I specify the path of the folder for a user to not access, e.g (X:\SPC-Matthew's Folder) and i have the ALLOW box unchecked, so it is DISALLOW, but on the user i set the rule for, i can still see the X:\SPC-Matthew's Folder folder and access all of it's contents.
And just in case i did both a rule with a / and a \ because im not sure on the indexing of them and their respective meaning, either doesn't work though.
![image](https://user-images.githubusercontent.com/67557965/153208107-1bd0a0b3-700a-4c50-90d8-e2d1eaea787b.png)

**Reply by ramiresviana** [reply]
Filebrowser does not handles drive letters. All paths must be relative to the application root. By default the root is the current working directory of the process.

**Reply by JustMentalMatt** [reply]
oh... filebrowser runs in a different drive to the files shared, im guessing i would have to move filebrowser to that drive, then put all of those files into the filebrowser folder?> or is there an easier way? would a shortcut work?- like i put a shortcut of the folder on the other drive in the root of filebrowser and then block that one.... let me try.

**Reply by JustMentalMatt** [reply]
UPDATE: nothing above works, putting the filebrowser files in the same drive as my files and putting my files in the root of filebrowser has no effect, nor does that thing with the shortcuts do anything :(

Im on windows if that matters at all.

**Reply by JustMentalMatt** [reply]
NVM i got it working with the REGEX thing. i just didnt know what regex was or how to use it i just randomly thought of trying [foldername].*  and it works magically! 
Although i do have a folder called $RECYCLE.BIN and it doesnt appear on the physical drive, in windows, (even with hidden files on), but it appears in the filebrowser.

OK my issue is sorted, just moved all folders into a different folder and made that folder root, so the recycle bin one doesn't get picked up by filebrowser!
